demisto_sdk/commands/content_graph/content_graph_builder.py

```diff
@@ -61,7 +61,7 @@ def create_graph(self) -> None:
         self.content_graph.create_indexes_and_constraints()
         self.content_graph.create_nodes(self.nodes)
         self.content_graph.create_relationships(self.relationships)
-        self.content_graph.validate_graph()
+        # self.content_graph.validate_graph()
```

Comment:
Please add this back - if there are specific validations issues handle them specifically